pkgs/top-level/all-packages.nix

```diff
@@ -31373,7 +31373,7 @@ with pkgs;
   firefox-esr-102 = wrapFirefox firefox-esr-102-unwrapped { };
   firefox-esr-115 = wrapFirefox firefox-esr-115-unwrapped { };
 
-  firefox-bin-unwrapped = callPackage ../applications/networking/browsers/firefox-bin {
+  firefox-bin-unwrapped = callPackage (if stdenv.isDarwin then ../applications/networking/browsers/firefox-bin/darwin.nix else ../applications/networking/browsers/firefox-bin/linux.nix) {
```

Comment:
Does the wrapper work on Darwin? Do we need it?

Reply:
The darwin conditional should be for firefox-bin then.

firefox-bin-unwrapped is more of a plumbing thing. I don't know why it's exposed. It doesn't hurt I guess but it's not intended to be something a user would want to install I think.

Reply:
Updated :+1: